onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -1031,6 +1031,7 @@ struct ProviderLibrary {
     }
 
     provider_ = PGetProvider();
+    provider_->Initialize();
```

Comment:
>Initialize [](http://example.com/codeflow?start=15&length=10)

not too clear on what the error handling strategy is in ProviderLibrary::Get(). Initialize() may throw, but on the other hand the failure statuses above are just logged. should clients expect both returned nullptr and exceptions if an error occurs?